docs/heasarc/heasarc.rst

```diff
@@ -189,6 +189,20 @@ that can be queried.
 The returned table includes both the names and a short description of each 
 mission table.
 
+Using alternative HEASARC servers
+-------------------------------
+
+It is possible to set alternative locations for HEASARC server. One such location
+is hosted by `INTEGRAL Science Data Center <https://www.isdc.unige.ch/>`_, and has further, 
+more recent, tables relevant for INTEGRAL spacecraft.
```

Comment:
suggestion: add an example grabbing one of those new tables to show users why they might want to use this mirror

Reply:
good idea, added. And I also added example for requesting time range in general case (as well as in our tables - it is critical there).